api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -125,7 +125,22 @@ default List<WorkspaceResponse> toApiWorkspaceResponseList(
   Workspace onlyForMappingRecentWorkspace(DbWorkspace dbWorkspace);
 
   @Mapping(target = "workspace", source = "dbWorkspace")
-  RecentWorkspace toApiRecentWorkspace(DbWorkspace dbWorkspace, WorkspaceAccessLevel accessLevel);
+  RecentWorkspace toApiRecentWorkspace(
+      DbWorkspace dbWorkspace,
+      WorkspaceAccessLevel accessLevel,
+      FeaturedWorkspaceService featuredWorkspaceService);
+
+  @AfterMapping
+  default void setFeaturedWorkspaceCategory(
+      @MappingTarget RecentWorkspace recentWorkspace,
+      DbWorkspace dbWorkspace,
+      WorkspaceAccessLevel accessLevel,
+      FeaturedWorkspaceService featuredWorkspaceService) {
```

Comment:
I'll check if we can start including the featured category info at dbworkspace level